Makefile

```diff
@@ -164,7 +164,7 @@ vet:
 tidy:
 	@echo "go mod tidy"
 	GO111MODULE=on go mod tidy
-	git diff --quiet go.mod go.sum
+	git diff --quiet go.mod go.sum api config controllers manifests pkg test
```

Comment:
I don't think this is the change I want to see. `tidy`should only check the go module dependency.

Something like https://github.com/pingcap/tidb-operator/blob/master/hack/verify-all.sh is better

Reply:
Done.  PTAL